infrastructure/main.tf

```diff
@@ -40,7 +44,8 @@ module "send-letter-service" {
     LETTER_TRACKING_DB_USER_NAME  = "${module.db.user_name}"
     LETTER_TRACKING_DB_PASSWORD   = "${module.db.postgresql_password}"
     LETTER_TRACKING_DB_NAME       = "${module.db.postgresql_database}"
-    FLYWAY_URL                    = "jdbc:postgresql://${module.db.host_name}:${module.db.postgresql_listen_port}/${module.db.postgresql_database}"
+    LETTER_TRACKING_DB_CONN_OPTIONS = "${local.db_connection_options}"
```

Comment:
Don't want to be that guy, but I saw in different PRs people like the vertical alignment in terraform files